src/components/EntityInfo/EntityMenu.vue

```diff
@@ -41,6 +41,34 @@
       </v-tooltip>
     </span>
 
+    <!-- Amalgamate -->
+    <span v-if="isBusiness && !isHistorical">
+      <v-tooltip
+        top
+        content-class="top-tooltip"
+        transition="fade-transition"
+        :disabled="!isAdminFrozen && !isPendingDissolution && !amalgamateTooltipText"
```

Comment:
I think you only need the last condition. This would also reduce duplicate code.

Reply:
Awesome catch!

Reply:
Looks good, thanks.